app/js/views/reminders.jsx

```diff
@@ -117,6 +119,21 @@ export default class Reminders extends React.Component {
     this.setState({ reminders });
   }
 
+  onServiceworkerChange() {
+    if (this.speechController.idle) {
+      location.reload();
+    } else {
+      const message =
+        'The application has just been updated. '+
+        'You may experience random issues until the app is reloaded. ' +
+        `We'll reload the app automatically soon.`;
```

Comment:
> With template strings you can have multi line strings.

I know but they keep the indentation inside as spaces, so this makes some ugly things :( (maybe in this case we don't care though, because they're feeding a HTML element)

Yeah maybe I can look at the wording. "Project Cue is updating and should restart shortly" ?